cppseminar/presentation/Pages/Admin/Submissions/List.cshtml

```diff
@@ -10,12 +10,24 @@
 <h3>Submissions</h3>
 <th:div asp-validation-summary="All"></th:div>
 <div>
-    <th:form method="get">
-        <label>
+    @{
+        int[] pages = new int[Model.numberOfPages];
+        for(var i=0; i < Model.numberOfPages; i++){
+            pages[i] = i;
+        }
+    }
+    <br>
+    <th:form method="get" id="pageForm">
+    <label>
+         <label>
             Filter by user
             <th:select asp-for="SelectedUser" asp-items="@Model.Users" onchange="this.form.submit();"></th:select>
         </label>
+        Page number
+        <th:select asp-for="PageNumber" asp-items="@(new SelectList(pages))" onchange="this.form.submit()"></th:select>
```

Comment:
I think we can thrash the code above to create pages array, just use here Enumerable.Range(0, Model.numberOfPages).ToArray(); (maybe it will even work without ToArray, which is of course preffered)